portfolio/src/main/webapp/jquery_functions.js

```diff
@@ -1,31 +1,59 @@
 /*
 * Purpose: contains various JQuery functions that listen for events on main portfolio page
 */
+// For each individual background img, width is 2016px height is 1512px
+const BACKGROUND_IMG_WIDTH = 2016;
+const BACKGROUND_IMG_HEIGHT = 1512;
+const VERT_PADDING = 40;
+const HORZ_PADDING = 20;
+const SPRITE_IMG_ROWS = 2;
+const SPRITE_IMG_COLS = 3;
+const NUM_IMAGES = 5;
 
-let img_arr = new Array(5);
-img_arr = ["../images/mountbierstadt.jpg", 
-           "../images/mountyale.jpg",
-           "../images/gardenofgods.jpg",
-           "../images/lakedillon.jpg",
-           "../images/mountelbert.jpg"
-           ];
 labels = ["the summit of Mount Bierdstadt.", 
           "the summit of Mount Yale.", 
+          "the summit of Mount Elbert, the tallest point in Colorado.",
           "the Garden of the Gods.", 
-          "Lake Dillon, seen from the vantage point of driving along Dillon Dam road.",
-          "the summit of Mount Elbert, the tallest point in Colorado."];
+          "Lake Dillon, seen from the vantage point of driving along Dillon Dam road."];
 
+profile_img_paths = ["images/noogler.gif",
+                     "images/zoebakercircle.png"
+                    ];
+
```

Comment:
There's now a lot of stuff in this file. Does it make sense to move some of the image constants/methods to their own "background_image.js" file and have that referenced/used by this one?

Reply:
Created a background_image.js file, and migrated everything into helper functions + 1 driver function.